### DISCUSSION TITLE
RQ2-viz

### DISCUSSION THREAD

**Discussion by anika-asthana** [ohtap/ohtap#41] [General]
NOTE: This is just to test out the correct way to upload visualizations. 

Date of Chart Creation: 4/5/2021
Link: https://github.com/ohtap/ohtap/blob/master/Research%20Question%202/Productivity%20and%20Frequency%20-%20Keywords%20(1).ipynb 
Data:  
[event_extents_with_nodes.csv](https://github.com/ohtap/ohtap/files/6316315/event_extents_with_nodes.csv)

<img width="723" alt="Screen Shot 2021-04-15 at 1 16 50 AM" src="https://user-images.githubusercontent.com/72761022/114837810-f404cb00-9d88-11eb-8acc-0a7db785e4dd.png">

@ebf77

**Comment by anika-asthana** [comment]
NOTE: These are preliminary - they are made with old data
Date of chart creation 4/19/21
Link: http://localhost:8888/notebooks/Productivity%20and%20Frequency%20-%20Keywords.ipynb
Data: http://localhost:8888/notebooks/Productivity%20and%20Frequency%20-%20Keywords.ipynb 

<img width="545" alt="Screen Shot 2021-04-19 at 3 08 41 PM" src="https://user-images.githubusercontent.com/72761022/115309438-23b52980-a121-11eb-9307-25840c064701.png">


<img width="555" alt="Screen Shot 2021-04-19 at 3 09 10 PM" src="https://user-images.githubusercontent.com/72761022/115309482-3596cc80-a121-11eb-82ac-ea69188ab6ab.png">

@ebf77

**Comment by anika-asthana** [comment]
CREATED BY Anika Asthana 
Date of chart creation: 4/29/21 

Description: 
This table displays the frequency of sexual harassment keyword hits within sexual harassment events by year of interview. Because the hits were all variations on molest, sex* harass*, harass, and pinch, I consolidated the keyword hits into four columns. In this case, the need for an 'other' column was not necessary. 

The data is from the master document released in Jan 2021, and the most recent metadata sheets for Interviewer and Interviewee. 

Link: https://github.com/ohtap/ohtap/tree/master/Research%20Question%202 
Data:https://github.com/ohtap/ohtap/tree/master/Research%20Question%202  

@ebf77 
Can you let me know what you think of the visualization and my presentation of it? 
I would like your feedback on how to best consolidate the years. I think that doing one row for every two years would be helpful. 


<img width="687" alt="Screen Shot 2021-04-30 at 4 17 23 AM" src="https://user-images.githubusercontent.com/72761022/116688046-f95a3c00-a96a-11eb-8785-12c4826e71d2.png">

<img width="625" alt="Screen Shot 2021-04-30 at 4 16 56 AM" src="https://user-images.githubusercontent.com/72761022/116688001-e9425c80-a96a-11eb-81bd-cfea9cdbf3c2.png">

**Comment by ebf77** [comment]
Hi Anika,
I’m seeing the table with raw data (possibly cut off at the bottom, unless Gartrell is the latest), but I don’t the visualization (nor on Github). And I’m curious that there are so few sexual harassment coded interviews conducted 1976-2001. Jade has worked on interview year and produced a table and graph that show around 16 pre 2000. See line 51 in this discussion: https://github.com/ohtap/ohtap/blob/Research_Question_1/Research_Question_1/Harassment_paper_work.ipynb. Let me know if I’m missing something!
Thanks,
Estelle

**Comment by anika-asthana** [comment]
Hi Dr. Freedman,

Gartrell is the last row. I'm not sure why it was cut off, but that
won't happen again.
When we were on zoom, I thought we discussed getting a table for you first
and then deciding how to clump the years and producing a visualization.

I took a look at Jade's data and I think that we used different initial
datasets.
I think there are so few pre-1976 interviews in mine because I'm using
Yibing's cleaned metadata set from late 2020. I reached out to her last
week about getting me the code to making a new one, but she hasn't
responded.

I'm going to try using the dataset that Jade used and produce an updated
table with more data!
Also, how do you think I should transform this table into a visualization?
Would you prefer to see it as a bar or a line graph?

Thanks,
Anika Asthana

**Reply by anika-asthana** [reply]
@ebf77 
Hi Dr. Freedman - 

I just realized that I likely forgot to send this to your email. Can you let me know what would be best for you to have before the meeting on the 25th? 
Also, Yibing said she would send me a new dataset this week, so hopefully the data will look more accurate! 

Thanks, 
Anika Asthana

**Comment by ebf77** [comment]
Hi Anika,
I’m not sure what you are referring to – what did you forget to send? I think we spoke since you sent the table. If you are using NVivo produced data, there is a new master that Natalie uploaded a few days ago. Jade is working with it now.
Generally I need to see the table first, then think about display. I’m overbooked with other project tasks right now so can’t guide you further at present. If you have a new table with the updated data, let me take a look.
Best
Estelle

**Comment by anika-asthana** [comment]
CREATED BY Anika Asthana
Date of chart creation 5/24/21
Link: https://github.com/ohtap/ohtap/blob/master/Research%20Question%202/SH%20Keyword%20Frequency%20and%20pre:post%201991%20tables.ipynb
Data: https://github.com/ohtap/ohtap/blob/master/Research%20Question%202/updated_0510.csv

Full Table: 
This table shows the frequencies of sexual harassment keywords within interviews conducted from 1972 to 2018. All column titles, such as 'Molest' account for all of the variations of the word, such as 'molested, molesting, molest.' The 'Other' category contains hits for all variations of seduction, cat call, peeping tom, and expose himself. 

<img width="635" alt="Screen Shot 2021-05-24 at 11 53 59 PM" src="https://user-images.githubusercontent.com/72761022/119452519-5091c900-bceb-11eb-885b-58999058456b.png">


Pre and Post 1991: 
This table shows the frequencies of sexual harassment keywords from before and after 1991. I am not sure why the count for ravish is so high, or why the count of hanky-panky is much higher post-1991. 
<img width="484" alt="Screen Shot 2021-05-24 at 11 54 13 PM" src="https://user-images.githubusercontent.com/72761022/119452545-58ea0400-bceb-11eb-8af9-cf9697884505.png">

@ebf77

**Comment by anika-asthana** [comment]
CREATED BY Anika Asthana
Date of chart creation 6/12/21
Link: https://github.com/ohtap/ohtap/blob/master/Research%20Question%202/SH%20Keyword%20Frequency%20and%20pre:post%201991%20tables.ipynb
Data: https://github.com/ohtap/ohtap/blob/master/Research%20Question%202/updated_0510.csv

NOTE: This table is a revised version of the table created on 5/24/21. It resolves the high ravish counts. The ravish regex (rav,ravish,ravished) was returning hits for words such as 'aggravate,' and no hits for 'ravish' or 'ravished' within events coded as rape or sexual harassment. 
 
Full Table:
This table shows the frequencies of sexual harassment keywords within interviews conducted from 1972 to 2018. All column titles, such as 'Molest' account for all of the variations of the word, such as 'molested, molesting, molest.' The 'Other' category contains hits for all variations of seduction, cat call, peeping tom, and expose himself. 

<img width="619" alt="Screen Shot 2021-06-12 at 6 18 55 PM" src="https://user-images.githubusercontent.com/72761022/121792515-a818ab80-cbaa-11eb-84d8-9fa32df45a85.png">

Pre and Post 1991:
This table shows the frequencies of sexual harassment keywords from before and after 1991. I have resolved the issues for ravish and hanky panky. There were no proper hits for any variation of ravish, but there were proper hits for hanky-panky. 

<img width="460" alt="Screen Shot 2021-06-12 at 6 19 27 PM" src="https://user-images.githubusercontent.com/72761022/121792521-ba92e500-cbaa-11eb-8d06-e449db432cc4.png">

@ebf77

**Comment by ebf77** [comment]
Hi Anika,

Thanks for sending these frequencies. You don’t have to do visualizations, but do supply the percentages for the rows in the second table only, pre/post 1991 table.

But first, the numbers in these tables don’t look right, compared to the data at the end of your Github link. Look at are the high frequencies for “abuse” in the years 2004, 2013, and 2018? Some other years also don’t match with the numbers in the table at the first Link. Could there be an error similar to “aggravate”? Is this the most recent table below, or am I missing something! Please double check, perhaps ask Yibing to go over the work with you.

Best
Estelle

**Reply by anika-asthana** [reply]
Hi Dr. Freedman, 

I worked with Yibing to resolve the issues. She gave me an updated, finalized dataset. The numbers are significantly smaller because the previous model I was using to calculate the frequencies wasn't de-duplicated. I was just counting the same words over and over again. 

Additionally, the 'abuse' and 'sexual abuse' keywords and subsequent variations are part of the 'rape and sexual abuse' cluster. The abuse numbers were so high on the chart I pushed on 6/12/21 because it was returning hits for abuse in the context of work or emotions, in addition to sexual assault. The Rape keyword cluster now just contains all variations of 'sexual abuse,' because 'abuse' was  unproductive. 

I also found that the 'Clarence Thomas' and 'Anita Hill' keywords were more productive than I initially realized. I added the data to it's own column instead of keeping it in 'Other.' 

CREATED BY Anika Asthana
Date of chart creation 6/22/21

Link: https://github.com/ohtap/ohtap/blob/Research_Question_2/Research%20Question%202/SH%20Keywords%20and%201991%20tables%200622.ipynb

Data: texts_nodes_by_interviewee_0618.csv
https://drive.google.com/file/d/1ViSLQ7y3fuuSI7EKn-rRlzOQsnaM95OO/view 

Full Table:
This table shows the frequencies of sexual harassment keywords within interviews conducted from 1972 to 2018. All column titles, such as 'Molest' account for all of the variations of the word, such as 'molested, molesting, molest.' The 'Other' category contains hits for all variations of seduction, cat call, peeping tom, hanky-panky, felt me up, and expose himself. The 'SH' column stands for 'Sexual Harassment.' 


<img width="602" alt="Screen Shot 2021-06-22 at 11 39 37 AM" src="https://user-images.githubusercontent.com/72761022/122981378-878ce600-d34e-11eb-95ac-ff11fe9adfea.png">


Pre/Post 1991: 
This table shows the frequency of sexual harassment keywords in sexual harassment events in a way that compares them before and after 1991. The parallel between Clarence Thomas and Anita Hill column and the Sexual Harassment column is particularly interesting! 

<img width="467" alt="Screen Shot 2021-06-22 at 11 39 21 AM" src="https://user-images.githubusercontent.com/72761022/122981337-7e037e00-d34e-11eb-91aa-65f7c929882c.png">


Pre-and-post 1991 Percentage Table: 
This table shows the percentage of sexual harassment keywords in sexual harassment coded events. The percentages were calculated by dividing each pre/post sub-section by the total number of hits from 1974-2018 for each keyword. 

<img width="479" alt="Screen Shot 2021-06-22 at 12 01 18 PM" src="https://user-images.githubusercontent.com/72761022/122984163-90cb8200-d351-11eb-8a00-f6a3f14cd960.png"> 

@ebf77

**Comment by ebf77** [comment]
Thanks so much Anika.

At first I was struck by how few accounts come from interviews conducted before 1992, suggesting the importance of the Thomas/Hill testimonies (use last names for them below, not first names). However, there are NO HITS listed from 1992 to 2002! That suggests a rather delayed effect. Are you sure that is correct, no interviews conducted in these years had any SH coding?

I would like to try different periods, 4 of them, around 11 years each: 1974-1984, 1985-1995, 1996-2006, and 2007-2018.

But first, a few questions about the methods and labelling:

Could you clarify (and add a column to the tables and the total number to the explanations): what is the total number of SH coded interviews per year? It looks like only 13 interviews pre 1992 were coded with SH. Is that correct? Or is it 13 instances of these terms in X number of interviews? (I’m thinking about the interview as a unit of measurement because if a long account (or two) of SH in one interview mentioned a keyword many times, it seems to weight them more heavily. )

Can you confirm that the basis for the percent is keywords, for all years. Please indicate the total numbers of keywords found for each period in a right hand column. Then you can also give the percent by rows (distribution of terms per period)  “The percentages were calculated by dividing each pre/post sub-section by the total number of hits from 1974-2018 [WHAT IS THAT TOTAL] for each keyword.”

No rush on this task – it will evolve further, I’m sure.

Many thanks!
Estelle

**Reply by anika-asthana** [reply]
Hi Dr. Freedman, @ebf77 

I went camping last week and was out of service, so I will get started on this! 
I'll let you know what progress I make as I go along. 

Thanks, 
Anika

**Reply by anika-asthana** [reply]
Hi Dr. Freedman, 

I think I figured out why it seems like there are fewer or no coded interviews. Some of the results for specific years aren't getting counted as float values, so I am going to manually change that and re-do the tables. 

Sorry it's taking so long! 
Anika  

@ebf77

**Comment by ebf77** [comment]
Dear Anika,

I hope that you are well, and I suspect that you are preparing to leave for Princeton soon! I wanted t thank you for all of you help with OHTAP over the past year and a half. If you have any further work to submit, please let me know in the next week. We will keep you posted of the outcomes of our efforts.

Gratefully,
Estelle


Estelle B. Freedman
Robinson Professor in U.S. History
Stanford University
Faculty Fellow, Center for Advanced Study in the Behavioral Sciences
